packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Build an API service integration
+excerpt: Learn how to build and register an API service integration to the Okta Integration Network.
+layout: Guides
+---
+
+<ApiLifecycle access="ea" />
+
+This guide explains how to build and submit an API service integration to the Okta Integration Network (OIN) for review and publishing.
+
+---
+
+**Learning outcomes**
+
+* Learn how to build an API service integration that can be published in the OIN.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+   > **Note:** Your org needs to have the API Service Integration feature enabled. Contact your Okta account team to enable the feature.
```

Comment:
If it goes out with the Nov release, we should have self-service EA at that point. They can turn it on in the console rather than contacting us. I think the feature name is the same as what you have here.